js/components/controlBar.js

```diff
@@ -638,8 +642,25 @@ var ControlBar = React.createClass({
       }).bind(this),
 
       "multiAudio": (function (alignment) {
+        var closeCaptionsList =  [];
```

Comment:
Can we move this block of code to a function that returns these lists? This section of the code is already very crowded